DakeAndDevileCorps/Screens/Store/VC/StoreDetailViewController.swift

```diff
@@ -7,13 +7,133 @@
+    @IBOutlet weak var storeDetailTableView: UITableView!
+    
+    var storeInformations: [StoreInformation] = []
+    var section2DataSource = [ProductTableViewCellModel]()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        storeDetailTableView.dataSource = self
+        storeDetailTableView.separatorStyle = UITableViewCell.SeparatorStyle.none
+        storeDetailTableView.rowHeight = UITableView.automaticDimension
+        storeDetailTableView.estimatedRowHeight = 100
+        if #available(iOS 15.0, *) {
+            storeDetailTableView.sectionHeaderTopPadding = 0
+        }
+        
+        storeName.text = "알맹상점"
+        storeName.font = UIFont.boldSystemFont(ofSize: 22)
+        
+        initStoreInformationData()
+    }
+    
+    func initStoreInformationData() {
+        section2DataSource = [
+            .product(productName: "주방세제", isSeperated: true),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .product(productName: "세탁세제", isSeperated: false),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .product(productName: "섬유유연제", isSeperated: false),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .product(productName: "기타세제", isSeperated: false),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .product(productName: "헤어", isSeperated: false),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .product(productName: "스킨", isSeperated: false),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .product(productName: "바디", isSeperated: false),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .product(productName: "생활", isSeperated: false),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .product(productName: "문구", isSeperated: false),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .product(productName: "애견", isSeperated: false),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원"),
+            .product(productName: "기타", isSeperated: false),
+            .item(itemName: "인블리스 세탁세제", itemPrice: "1g = 4원")
+        ]
     }
-
-
 }
 
+extension StoreDetailViewController: UITableViewDataSource {
+    
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return 2
+    }
+    
+    private func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
+        switch section {
+        case 0:
+            return nil
+        case 1:
+            return nil //collectionView
+        default:
+            return nil
```

Comment:
여기는 나중에 HeaderView가 추가되는 걸까요?

Reply:
네 그렇습니다. 여기서 궁금한 점이 header가 없는 section1에서 nil을 return해줬는데 공간이 잡혀서 section1의 height를 0으로 return 해주고 있습니다. 원래 그런 걸까요?

Reply:
이후 `case 1`만 사용하실 거라면, `case 0`의 경우와 `default`의 경우가 같으니 `case 0`은 생략해도 좋을 것 같습니다.

```suggestion
        switch section {
        case 1:
            return nil //collectionView
        default:
            return nil
```

Reply:
> 네 그렇습니다. 여기서 궁금한 점이 header가 없는 section1에서 nil을 return해줬는데 공간이 잡혀서 section1의 height를 0으로 return 해주고 있습니다. 원래 그런 걸까요?

원래 이상하게 그러더라구요.. 전에는 0으로 해도 크기가 잡혀서 0.5로 잡은 적도 있었습니다..💩

Reply:
[5e6a8ce](https://github.com/DeveloperAcademy-POSTECH/MC3-Team4-DakeAndDevileCorps/pull/22/commits/5e6a8cebef8a9e197d437943eef27e51e937fcdd)에서 스누피가 피드백 주신 내용 반영하였습니다!

@YoonAh-dev  아주💩이군요..!